drivers/sensor/ccs811/Kconfig

```diff
@@ -60,6 +60,16 @@ config CCS811_TRIGGER_OWN_THREAD
 
 endchoice
 
+config CCS811_RESET
+	bool "Use gpio for reset"
+	depends on GPIO
+	default n
+
+config CCS811_WAKE
+	bool "Use gpio for wake"
+	depends on GPIO
+	default n
```

Comment:
this makes it impossible to have different configurations between instances. I think the best solution is to:

1. have a single property that specifies such gpio, e.g. `gpios = <>;`
2. have a DT property that allows to set the functionality for such gpio, e.g. `gpio-function = "wake";`

Reply:
I see the problem with the current solution now, but I still have some questions:

What mechanism should be used to enable `struct gpio_dt_spec reset_gpio` and `struct gpio_dt_spec wake_gpio` once these appear in at least one instance of the dts, respectively?

How do we differentiate instances from each other inside the shared driver functions?

Reply:
The best solution I've come across for the second question is to check `reset_gpio.port`. However, I'm still not sure what's the best solution for the first question.